packages/sync-repo-settings/src/required-checks.json

```diff
@@ -149,6 +135,32 @@
           "Kokoro - Test: Linkage Monitor",
           "cla/google"
         ]
+      },
+      {
```

Comment:
Why haven't these two come along for the ride?

Reply:
Spanner is currently blocked because it requires an extra environment variable for unit tests we haven't set yet. Firestore is just not merged yet.

Reply:
Can I get an issue to track both of these?  Putting high value here on consistency.

Reply:
#547 #548 #549 Created.